includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -87,7 +124,7 @@ public function enqueue_editor_assets() {
 
 		if (
 			( ! empty( $excerpt_roles ) && empty( array_diff( $user_roles, $excerpt_roles ) ) )
```

Comment:
As mentioned above, I'd move this check within the new `user_can_access` method

Reply:
@dkotter, The fix for this is now ready for review. 

cc @jeffpaul